tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -68,6 +68,35 @@ stages:
           run_end_to_end: false
           container: flink-build-container
           jdk: jdk8
+      - job: docs_404_check # run on a MSFT provided machine
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          # Skip docs check if this is a pull request doesn't contain documentation change
+          - bash: |
+              source ./tools/azure-pipelines/build_properties.sh
+              contains_docs_pullrequest
+              if [[ "$?" == 0 ]] ; then
+                echo "##[debug]This is a pull request doesn't contain documentation. Skipping docs check."
+                echo "##vso[task.setvariable variable=skip;]1"
+              else
+                echo "##[debug]This is a regular CI build. Continuing ..."
```

Comment:
```suggestion
```
I'd think this just adds confusion.

Reply:
I rephrased both messages. Since this is scoped to the docs_404_check / "Check if PR contains docs change", it will be clear what's going on. It won't confuse people checking the regular build output, because it won't show there.